src/sections/InfoCardGrid.scss

```diff
@@ -30,16 +25,17 @@
   @screen md {
     @apply flex;
     @apply flex-wrap;
+    column-gap: 1rem;
   }
 
   .info-card {
     & > :not(.info-card__title) {
-      @apply text-sm;
+      font-size: var(--info-card-font-size);
     }
 
     @screen md {
-      width: calc(50% - 1rem);
-      margin-right: 1rem;
+      width: calc(50% - 0.5rem);
```

Comment:
This is a known visual adjustment / improvement from Detroit. It removes unnecessary right-hand padding. You'll see a slight difference in the story.